src/test/resources/com/cloudbees/jenkins/plugins/bitbucket/client/payload/2.0-repositories-amuniz-test-repos-pullrequests-1-commits_fields_values.hash_2Cvalues.author.raw_2Cvalues.date_2Cvalues.message_pagelen_1.json

```diff
@@ -1,10 +0,0 @@
-{
-  "values": [{
-    "date": "2018-09-21T14:53:12+00:00",
-    "message": "[CI] Release version 1.0.0",
-    "hash": "bf0e8b7962c024026ad01ae09d3a11732e26c0d4",
-    "author": {
-      "raw": "Builder <no-reply@acme.com>"
-    }
-  }]
-}
```

Comment:
Why was this deleted? I assume this was deleted to make a test pass?

Reply:
Hi, it was probably a mistake, I didn't try to delete it...